tutorials/optimization/cpu_optimization.rst

```diff
@@ -165,16 +169,20 @@ code, consider moving those calculations to a faster language.
 GDScript
 ~~~~~~~~
 
-GDScript is designed to be easy to use and iterate, and is ideal for making many
-types of games. However, ease of use is considered more important than
-performance, so if you need to make heavy calculations, consider moving some of
-your project to one of the other languages.
+:ref:`toc-learn-scripting-gdscript` is designed to be easy to use and iterate,
```

Comment:
There's an error on this one but I don't get why:
```
/home/runner/work/godot-docs/godot-docs/tutorials/optimization/cpu_optimization.rst:172:undefined label: toc-learn-scripting-gdscript (if the link has no caption the label must precede a section header)
```

Reply:
I tried adding explicit names for TOC references, does it work now?